int32 traj_id
int32 traj_order
float64 start_t
float64 duration
float64[] coeff
float64[] seg_t